src/app/materials/materials.module.ts

```diff
@@ -23,13 +26,15 @@ const COMPONENTS = [
   MatButtonModule,
```

Comment:
This records are placed in `const COMPONENTS`, but they aren't components.

Reply:
renamed to MODULES